DEPLOYNOTES.rst

```diff
@@ -1,6 +1,14 @@
 Deploy Notes
 ============
 
+2.7
+---
+
+- To avoid an ImproperlyConfiguredError in Django 2.2, media must now be stored
+  outside static/. Web servers (Apache) will need to be reconfigured to look 
+  for media in a new location, and a redirect should be created so that links 
+  pointing to /static/media/ are not broken.
+
```

Comment:
I don't think it makes sense to mention the improperly configured error; I think this is already how we should have been doing things and this app just got missed somehow.  Also, unless I've misunderstood, we're not moving where anything is stored, just what urls they are configured under for web access — right? Here's a suggested revision, does this seem clearer to you?

> As part of the upgrade to Django 2.2, media should no longer be served under the url for static site content. Apache should be reconfigured to serve media content at `/media`, and a permanent redirect should be added so that content previously available under `/static/media` will resolve to the new location.